.travis.yml

```diff
@@ -33,15 +33,14 @@ jobs:
         - env
         - bash <(curl -s https://codecov.io/bash)
     - stage: openapi-checks
-      language: node_js
-      node_js:
-        - 17
-        - node
+      services:
+        - docker
+      before_install:
+        - docker pull openapitools/openapi-generator-cli
```

Comment:
Isn´t there a risk that docker blocks the downloads? They have a very small limit for anonymous accounts

Reply:
There is. I thought we should be fine because of the following: If you check out the [insights tab of our Travis](https://app.travis-ci.com/github/RedHatInsights?tab=insights), we see an average total of **600 jobs per week** for all the repositories, while the rate limit is **100 pulls every 6 hours**.

But I admit I couldn't figure out what kind of pool is used for the runners, so I'm looking into ways to not be bitten by the rate limiting.

So yes, there is a risk hat docker blocks the downloads, but IMHO it won't affect us as much as the current issue with the maven repository.

weekly insights: ![image](https://github.com/RedHatInsights/insights-results-smart-proxy/assets/15891027/8966160d-4bda-44b9-8409-36a1fd78b709)

monthly insights: 
![image](https://github.com/RedHatInsights/insights-results-smart-proxy/assets/15891027/0c59f53e-d6b5-4732-8932-c74ec1ba8ab8)

Reply:
Thanks for the insights. I would still be worried, because I guess Travis instances share IP addresses, so it is not how much you download but how much everyone does. But if you think the benefits overcome the drawbacks, let's do it.